docs/source/release_notes/52_0.rst

```diff
@@ -0,0 +1,139 @@
+========
+52.0 LTS
+========
+
+The Avocado team is proud to present another release: Avocado version
+52.0, the second Avocado LTS version.
+
+Release documentation: `Avocado 52.0
+<http://avocado-framework.readthedocs.io/en/52.0/>`_
+
+When compared to the last LTS (v36), the main features introduced by
+this versions are:
+
+- TAP[2] version 12 support.
+- The download of assets in tests now allow for an expiration time.
+- Environment variables can be propagated into tests running on remote
+  systems.
+- The plugin interfaces have been moved into the
+  `avocado.core.plugin_interfaces` module.
+- A contrib script for running `kvm-unit-tests`.
+- Support for running tests in Docker container.
+- Introduction of the “Fail Fast” feature.
+- Special keyword `latest` for replaying previous jobs.
+- The introduction of a tool that generated a diff-like report of two
+  jobs.
+- New `avocado.utils.process.SubProcess.get_pid()` method.
+- The introduction of a `rr` based wrapper.
+- The `coredump` generation contrib scripts will check if the user
+  running Avocado is privileged to actually generate those dumps.
+- New `avocado.utils.linux_modules.check_kernel_config()`.
+- The automatic VM IP detection that kicks in when one uses `--vm-domain`
+  without a matching `--vm-hostname`, now uses a more reliable method
+  (libvirt/qemu-gust-agent query).
+- Set `LC_ALL=C` by default on sysinfo collection to simplify avocado
+  diff comparison between different machines.
+- List result plugins (JSON, XUnit, HTML) in avocado plugins command
+  output.
+- Multiplexer: it now defines an API to inject and merge data into the
+  multiplexer tree.
```

Comment:
Multiplexer actually does not exists (in the 36lts meaning). So I'd start with `Multiplexer` was replaced with `Varianter` plugging system with defined API to register plugins which generate test variants. For details I'd point to it's documentation as there were quite a few changes. On next line I'd mention that the old `--multiplex` argument which used to turn yaml files into variants is now handled by an optional plugin called `yaml_to_mux` and the `--multiplex` option is deprecated and people should start using `--mux-yaml` command instead (which behaves the same way).